en/airframes/airframe_reference.md

```diff
@@ -39,6 +39,34 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Dodecarotor cox
```

Comment:
@bkueng There doesn't appear to be any image for this vehicle. Do you think we can get a build guide or other information?